skydriver/database/schema.py

```diff
@@ -12,6 +13,21 @@
 StrDict = dict[str, Any]
 
 
+class ScanState(enum.Enum):
+    """A non-persisted scan state."""
+
+    # completed states
+    COMPLETED_BEFORE_SCANNING = enum.auto
+    COMPLETED_WITH_FINISHED_SCANNING = enum.auto
+    COMPLETED_WITH_UNFINISHED_SCANNING = enum.auto
+
+    # non-completed states
+    SCANNING_IN_PROGRESS = enum.auto
+    SCANNER_WAITING_FOR_CLUSTERS = enum.auto
+    CLUSTERS_WAITING_FOR_SCANNER = enum.auto
+    PENDING_STARTUP = enum.auto
```

Comment:
Can we get better documentation for this?  I can guess what some of them are, but it would be good to be even more verbose.  It's also not clear what's an error vs success.